Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -654,5 +654,29 @@ void JitArm64::GenerateCommonAsm()
   paired_store_quantized[30] = storeSingleS8Slow;
   paired_store_quantized[31] = storeSingleS16Slow;
 
-  GetAsmRoutines()->mfcr = nullptr;
+  // Input in X2, output in W3, clobbers X2-X5. X0 and X1 are not used, because this is called
```

Comment:
IIRC we start with the lower registers. Might you use the registers which are allocated at least? So it will be less likely to push/pop registers here.

Reply:
I'm wrong here. The lower registers are fine: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp#L327